scripts/wait-for-all-codebuild.ts

```diff
@@ -39,7 +39,13 @@ const getFailedJobIdsFromBatchId = async (cb: CodeBuild, batchId: string): Promi
 };
 
 const main = async () => {
-  const cb = new CodeBuild({ region: 'us-east-1' });
+  const cb = new CodeBuild({
+    region: 'us-east-1',
+    maxRetries: 10,
```

Comment:
this should help with throttling .
see https://github.com/aws/aws-sdk-js/blob/29888001e1c3f6f9deefaf69ac2b46c5cebe1b72/lib/util.js#L905 .
I.e. 
```


Still waiting for these jobs: ["build_linux"]
--
51 | /codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/protocol/json.js:80
52 | resp.error = util.error(new Error(), error);
53 | ^
54 | ThrottlingException: Rate exceeded
55 | at Request.extractError (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/protocol/json.js:80:27)
56 | at Request.callListeners (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
57 | at Request.emit (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
58 | at Request.emit (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/request.js:686:14)
59 | at Request.transition (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/request.js:22:10)
60 | at AcceptorStateMachine.runTo (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/state_machine.js:14:12)
61 | at /codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/state_machine.js:26:10
62 | at Request.<anonymous> (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/request.js:38:9)
63 | at Request.<anonymous> (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/request.js:688:12)
64 | at Request.callListeners (/codebuild/output/src1542285976/src/github.com/aws-amplify/amplify-cli/scripts/node_modules/aws-sdk/lib/sequential_executor.js:116:18) {
65 | code: 'ThrottlingException',
66 | '[__type]': 'See error.__type for details.',
67 | time: 2023-08-14T15:11:15.301Z,
68 | requestId: '4a6bf99b-253b-4784-9771-ec56f4bfc0a6',
69 | statusCode: 400,
70 | retryable: true
71 | }


```